install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -348,3 +348,4 @@ data:
   enable-well-known-identities: "false"
 {{- end }}
   enable-remote-node-identity: {{ .Values.global.remoteNodeIdentity | quote }}
+  enable-api-rate-limit: {{ .Values.global.enableAPIRateLimit | quote }}
```

Comment:
We should be adding this to `config` rather than `global`.

Reply:
I think given that this is 1.7, the precedent here is `global`, so I'll stick with that.